workers/unity/Packages/com.improbable.gdk.buildsystem/Configuration/SpatialOSBuildConfiguration.cs

```diff
@@ -62,24 +62,46 @@ private void ResetToDefault()
         {
             WorkerBuildConfigurations = new List<WorkerBuildConfiguration>
             {
+                new WorkerBuildConfiguration
                 {
-                    new WorkerBuildConfiguration
+                    WorkerType = "UnityClient",
+                    LocalBuildConfig = new BuildEnvironmentConfig { BuildOptions = BuildOptions.Development },
+                },
+                new WorkerBuildConfiguration
+                {
+                    WorkerType = "UnityGameLogic",
+                    LocalBuildConfig =
+                        new BuildEnvironmentConfig { BuildOptions = BuildOptions.EnableHeadlessMode },
+                    CloudBuildConfig = new BuildEnvironmentConfig
                     {
-                        WorkerType = "UnityClient",
-                        LocalBuildConfig = new BuildEnvironmentConfig { BuildOptions = BuildOptions.Development },
+                        BuildPlatforms = SpatialBuildPlatforms.Linux,
+                        BuildOptions = BuildOptions.EnableHeadlessMode
                     }
                 },
+                new WorkerBuildConfiguration
                 {
-                    new WorkerBuildConfiguration
+                    WorkerType = "AndroidClient",
+                    LocalBuildConfig = new BuildEnvironmentConfig
+                    {
+                        BuildPlatforms = SpatialBuildPlatforms.Android,
+                        BuildOptions = BuildOptions.Development
+                    },
+                    CloudBuildConfig = new BuildEnvironmentConfig
+                    {
+                        BuildPlatforms = SpatialBuildPlatforms.Android
+                    }
+                },
+                new WorkerBuildConfiguration
```

Comment:
probably shouldn't add this as a default. if people want to use mobile, they need to select it